api/client/clientModel.js

```diff
@@ -60,6 +59,46 @@ function updateDelete(id) {
     });
 }
 
+// FOR CLIENT INTAKE ROUTES
+
+const updatePhoneNumbersByClientId = (id, changes) => {
+  return db('phone_numbers').where({ client_id: id }).update(changes, '*');
+};
+
+const updateEmailByClientId = (id, changes) => {
+  return db('email_addresses').where({ client_id: id }).update(changes, '*');
+};
+
+const updateEducationHistoryByClientId = (id, changes) => {
+  return db('education_histories')
+    .where({ client_id: id })
+    .update(changes, '*');
+};
+
+const getGenderIdByClientId = async (id) => {
```

Comment:
This model function doesn't make sense. The genders table is a lookup table with (id, name) fields. The id is unique to the gender not the client so this will never return a value, or it will be random for the first couple clients with id ~ 1-10. 

You will need to join the genders table on the clients table where the clients.id = client_id. Then you can return the gender_id and name